### DISCUSSION TITLE
I just upgraded to last release from 0.7.0. How do I manage fonts now?

### DISCUSSION THREAD

**Discussion by Julianoe** [dompdf/dompdf#3083] [Q&A] [answered]
How do I import local hosted fonts in this new version?

**Comment by bsweeney** [comment, marked as answer]
At this point we recommend using the `@font-face` declaration. [load_font.php](https://github.com/dompdf/utils/blob/master/load_font.php) is still available from the utils project but I have not validated its functionality with more recent releases.

**Reply by Julianoe** [reply]
Ok I missed that point. Replacing the old font loading with `@font-face` worked well. Thank you.
